regex.py

```diff
@@ -0,0 +1,14 @@
+
+
+import re
+
+test_str = 'hi there 255,255,255,255'
+
+pattern = re.compile(r"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?),(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?),(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?),(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)")
```

Comment:
There is another improvement we could make to this regex:

The regex has a lot of repetitive sections (`(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?),` is repeated 3 times in the regex). Do you know of a way to simplify the regex to reduce the amount of repetition?

Reply:
it must be a way to solve the repeating issue but I'm not sure I've seen it. I try to solve this little issue.